### PR TITLE
Remove legacy runner events

### PR DESCRIPTION
**Pull request by arbulu89**
# Description

Remove legacy events usage and code around it like unused read models and their database configuration. The code around host connection settings is remove as well.

The legacy events are moved to a `legacy` folder, because they must exist in the code, as they are part of the stream, but they don't apply any change to the aggregate anymore.

## How was this tested?

Mostly test were removed. A new single test to check the legacy events don't change the aggregate was added

### CODE REVIEW

File: lib/trento/domain/cluster/cluster.ex [hunk 1]
```diff
@@ -510,6 +336,8 @@ defmodule Trento.Domain.Cluster do
     snapshot
   end
 
+  def apply(cluster, %legacy_event{}) when legacy_event in @legacy_events, do: cluster
```
Comment (arbulu89) [inline lib/trento/domain/cluster/cluster.ex:339]: This is the unique new piece of code

**Review by nelsonkopliku — APPROVED** [review]
image

**Review by fabriziosestito — CHANGES_REQUESTED** [review]
We need also to clean up `TrentoWeb.OpenApi.Schema.Cluster`, mega LGTM for the rest.

**Comment by arbulu89** [comment]
> We need also to clean up `TrentoWeb.OpenApi.Schema.Cluster`, mega LGTM for the rest.

Thanks @fabriziosestito good catch.
Updated

**Review by dottorblaster — APPROVED** [review]
LGTM

**Pull request merged**